src/Controller/ParticipantController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\Participant;
+use App\Model\Tournament;
+use App\Services\ParticipantService;
+use App\Services\TournamentService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ParticipantController extends AbstractController
+{
+    private ParticipantService $participantService;
+    private TournamentService $tournamentService;
+
+    public function __construct(ParticipantService $participantService, TournamentService $tournamentService)
+    {
+        $this->participantService = $participantService;
+        $this->tournamentService = $tournamentService;
+    }
+
+    /**
+     * @Route("/api/tournaments/{tournamentId}/participants", name="add_participant", methods={"POST"})
```

Comment:
si vous êtes en PHP >=8 vous pouver utiliser les attributes => https://symfony.com/blog/new-in-symfony-5-2-php-8-attributes